conda_concourse_ci/execute.py

```diff
@@ -421,7 +421,7 @@ def sourceclear_task(meta, node, config_vars):
 def graph_to_plan_with_jobs(
         base_path, graph, commit_id, matrix_base_dir, config_vars, public=True,
         worker_tags=None, pass_throughs=None, use_lock_pool=False,
-        use_repo_access=False, automated_pipeline=False, folders=None):
+        use_repo_access=False, automated_pipeline=False, branches=[], folders=None):
```

Comment:
better to have the default be None and have a check for that.  [Mutable default argument](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) can cause a number of issues.

Reply:
resolved in a48c5bc70bd1585bfce1dd4eb7c55bb1bd61d105